README.rst

```diff
@@ -53,6 +53,8 @@ If you would like to test a deployment of S-R using Docker for CI purposes, then
 #. Make sure that you have a recent version of Docker. This guide has been tested on version 19.03.5 (build 633a0ea838).
    :code:`make deps` will install the latest version if you do not already have a version of docker available.
    Full instructions available at `the Docker website <https://docs.docker.com/engine/installation/linux/ubuntulinux/>`_.
```

Comment:
Whilst we're at it, let's also update the Ubuntu installation link (https://docs.docker.com/engine/install/ubuntu/) and also include a link for installing Docker on CentOS (https://docs.docker.com/engine/install/centos/)